sdks/python/README.md

```diff
@@ -8,12 +8,19 @@ Python >= 3.6
 
 ## Installation
 
-To install the latest development version of the SDK, run the following:
+If you'd like to install the official releases of the SDK on PyPI, please run the following:
+```
+pip install argo-workflows==6.3.0rc2
```

Comment:
I think it would be `v3.3.0-rc1`?

Reply:
We are at rc2 now since we did not release rc1 SDKs successfully. See https://github.com/argoproj/argo-workflows/issues/7692.

Reply:
Oh are you referring to the 6 vs 3? See the versioning section I added below.

Reply:
I mean there is a hyphen

Reply:
there should be a hyphen below too

Reply:
The lack of hyphen is intentional to follow Python package versioning conventions: https://www.python.org/dev/peps/pep-0440/

However, the Argo Workflows version is missing a hyphen. I'll add.

Reply:
Just fixed and added a note on this.